### DISCUSSION TITLE
When the closed tab node has children tab node(s), ask to close child tabs

### DISCUSSION THREAD

**Discussion by rocketeeer** [Able-Bear-Software/tab-nodes-tree#67] [Ideas / Feature Request]
Version: 1.4.1

Issue: There doesn't seem to be an easy way to close all tabs in a group.  When closing a parent tab, there is no option to close child tabs.

Video:
https://github.com/user-attachments/assets/ccf615a5-47bb-4e34-8611-2924daf62423

Desired behavior: When closing a parent tab, ask if all child tabs should be closed as well.

**Comment by zeroarst** [comment]
@rocketeeer 
Thank you for reporting this. 
Ticket created: https://github.com/Able-Bear-Software/tab-nodes-tree/issues/68

**Reply by stefan123t** [reply]
Thanks @zeroarst for already creating a ticket, I think this is an essential one.
Otherwise you have to close each child-tab individually.

**Comment by Tab-Nodes-Tree** [comment]
Released in [1.5.0](https://github.com/Able-Bear-Software/tab-nodes-tree/discussions/84).
